spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature 'User Index', type: :feature do
+  scenario 'visiting the user index page' do
+    User.create(name: 'Tom', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png')
+    User.create(name: 'Ali', photo: 'https://www.kasandbox.org/programming-images/avatars/leaf-blue.png')
+
+    visit users_path
+
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Ali')
+  end
+
```

Comment:
- [ ] Dear Team, good job adding the current tests but I can see you are missing the test `asserts you can see the profile picture for each user.` Please add this one.